xbmc/windowing/osx/WinSystemOSX.mm

```diff
@@ -641,6 +663,12 @@ static void DisplayReconfigured(CGDirectDisplayID display,
 
 bool CWinSystemOSX::CreateNewWindow(const std::string& name, bool fullScreen, RESOLUTION_INFO& res)
 {
+  // find the screen where the application started the last time. It'd be the default screen if the
+  // screen index is not found/available.
+  const std::shared_ptr<CSettings> settings = CServiceBroker::GetSettingsComponent()->GetSettings();
+  m_lastDisplayNr = GetDisplayIndex(settings->GetString(CSettings::SETTING_VIDEOSCREEN_MONITOR));
+  NSScreen* screen = [NSScreen.screens objectAtIndex:m_lastDisplayNr];
```

Comment:
`NSScreen.screens` index out of bounds if `m_lastDisplayNr` was for a display that is now disconnected.